src/main/java/lotto/domain/Rank.java

```diff
@@ -1,14 +1,17 @@
 package lotto.domain;
 
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public enum Rank {
 	FIRST(6, 2_000_000_000),
-	SECOND(5, 1_500_000),
-	THIRD(4, 50_000),
-	FOURTH(3, 5_000),
-	INVALID(0, 0);
+	SECOND_BONUS(5, 30_000_000),
+	THIRD(5, 1_500_000),
+	FOURTH(4, 50_000),
+	FIFTH(3, 5_000),
+	MISS(0, 0);
 	private int matchCount;
 	private int prize;
```

Comment:
https://github.com/next-step/java-lotto/pull/3781#discussion_r1564614910

```suggestion
	private int matchCount;
	private int prize;
	private Bonus bonus;
```

상태로 값을 체크한다면, enum 클래스의 상태만으로도 체크하는 로직을 유추할 수 있을것 같아 피드백 드린부분이에요 ㅎㅎ
지금은 checkBonus 와 같이 세부 구현을 들여다봐야 2등만 체크하는 로직인지 확인이 가능한 구조니까요!
다양한 방식으로 구현하는 법을 경험하셨으면 하여 코멘트 드렸습니다 😄

Reply:
넵 이해했습니다 enum 클래스에서도 바로 유추할 수 있게 수정해보겠습니다! 리뷰해 주셔서 감사합니다!